.opencode/agents/silent-failure-hunter.md

```
@@ -0,0 +1,71 @@
+  Read: allow
+  Grep: allow
+  Glob: allow
+---
+
+# Silent Failure Hunter Agent
+
+You hunt for error-handling patterns that silently swallow failures, hiding bugs and making debugging impossible.
+
+## What to Check
+
+### 1. Silent Except Blocks (HIGH)
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)